.github/workflows/continous-integration.yml

```diff
@@ -255,7 +255,6 @@ jobs:
       with:
         path: ${{ env.CHOCO_CACHE_LOCATION }}
         key: ${{ runner.os }}-system-${{ hashFiles('system_test_dependencies/windows.txt') }}
-        restore-keys: ${{ runner.os }}-system
```

Comment:
so I think I misconfigured this one 🤦🏻 sounds like it should be a list instead of a single string... But anyway, it's ok to remove for now